modules/tile-converter/src/converter-cli.ts

```diff
@@ -99,6 +101,9 @@ function printHelp(): void {
   );
   console.log('--name [Tileset name]');
   console.log('--output [Output folder, default: "data" folder]');
+  console.log(
+    '--instant-nodes-writing [Keep created 3DNodeIndexDocument files on disk instead of memory. This option reduce memory usage but decelerates conversion speed]'
```

Comment:
```suggestion
    '--instant-node-writing [Keep created 3DNodeIndexDocument files on disk instead of memory. This option reduce memory usage but decelerates conversion speed]'
```